### PR TITLE
 Use bash base for ci-k8sio-audit

### PR DESCRIPTION
**Pull request by hh**
go 1.15, bq, gsutil, gcloud, jq, pr-creator

We don't currently have an image that includes the above.

This is an attempt to experiment before creating a more formal image.

### CODE REVIEW

File: config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml [hunk 1]
```diff
@@ -64,23 +91,27 @@ periodics:
         popd
 
         echo -n "Calculate github user, name, and email from token: " >&2
+        # curl https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
+        #   -L -o /usr/local/bin/jq && chmod +x /usr/local/bin/jq
+        echo $PATH
```
Comment (hh) [inline config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml:96]: Will remove next commit... will use alpine package instead

**Review by justaugustus — COMMENTED** [review]
@hh -- May I suggest building on top of (or adding to) [`releng-ci`](https://github.com/kubernetes/release/blob/master/images/releng/ci/Dockerfile)?

- it has a lot of what you need already
- we (RelEng) keep it reasonably up-to-date
- it's already being used for several of the RelEng (and some K8s Infra) jobs: https://cs.k8s.io/?q=releng-ci&i=nope&files=&repos=

**Comment by justaugustus** [comment]
Some background PRs:
- https://github.com/kubernetes/release/pull/1552
- https://github.com/kubernetes/release/pull/1679

**Comment by hh** [comment]
Will look at it in the morning! Thanks @justaugustus !

**Comment by justaugustus** [comment]
> Will look at it in the morning! Thanks @justaugustus !

For sure! And for the usual OSS line: "Contributions welcome (and encouraged)"
The goal for that image was to be general purpose enough that it could be used in a variety of places and it doesn't look like it'll take too much to get it to do what you need. :) 💕

**Comment by hh** [comment]
Updated pr-creator with some clarity with regards to the error message. Will look at this again next week.
https://github.com/kubernetes/test-infra/pull/20925

**Pull request closed without merging**